Add Total Weight Option to wc-cart-pdf

gianhunold opened the pull request.

gianhunold committed:
Add Total Weight to PDF Export

dkjensen commented on templates/cart-table.php:
```diff
@@ -134,6 +134,28 @@
+
+				if ($total_weight > 0):
+					// Convert grams to kilograms
+					$total_weight_kg = $total_weight / 1000;
```
@gianhunold We should not be assuming the weight is in KG. Can you update this to reflect native WooCommerce behavior when rendering the cart weight?

gianhunold committed:
Add AutoSwitch Unit Setting

gianhunold commented:
First of all, thanks for considering this new feature.
Yes, i get you. Some people may want grams and some may even use oz and lb. 

I now updated it to use the default woocomerce unit and added a extra setting that lets you convert to the bigger unit if the smaller unit is "exceeded".

What do you think about it?
